pkg/controller/node/node_controller.go

```diff
@@ -988,6 +989,7 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 			return err
 		}
 	}
+	ctrlcommon.UpdateStateMetric(ctrlcommon.MCCSubControllerState, "machine-config-controller-node", "Sync Machine Config Pool", pool.Name)
```

Comment:
Unlike the other metrics, I think this one will happen whenever a sync finishes? As in, even if nothing changes. So this could update quite often, without doing anything.

Would this make more sense to apply if a change has happened (so the controller is doing something)?

Same question applies to the controllers below as well